packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -7,9 +7,17 @@ public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(EndpointConfiguration endpointConfiguration)
         {
-            // NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. 
+            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
```

Comment:
How about also add the action for the user next to the TODO?
For example:

//TODO: Pick a persistence that meets your requirements. NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.